zvariant/src/dbus/de.rs

```diff
@@ -188,7 +188,13 @@ impl<'de, 'd, 'sig, 'f, #[cfg(unix)] F: AsFd, #[cfg(not(unix))] F> de::Deseriali
             .endian()
             .read_f64(self.0.next_const_size_slice::<f64>()?);
 
-        visitor.visit_f32(f64_to_f32(v))
+        if v.is_finite() && v > (f32::MAX as f64) {
```

Comment:
should this be `!v.is_finit()`? You need to update the test code btw.

Reply:
I'd really hope not. We want _finite_  (not inf or NAN) to compare against f32::MAX  in order to cause an error. 

If we wanted to be really serious, we'd check small f32 values against precision bits in the f64 number, but that would be slow for very little gain, if any.  

ie. what happens if the f64 float number has digits in the precision range that cannot be encoded as f32 and would cause "bad" rounding. Honestly, not worth making "proper" as the error would be one of those like "and then what do we do with it?" ( It probably means that whomever encoded it may not have used an f32...)

Reply:
Ah right. I just misread the `&&` as `||` for some reason. Thanks for explaining.